src/components/molecules/Help/Help.tsx

```diff
@@ -9,32 +9,61 @@ import Card from '../../organisms/Card';
 import { buttonStyle } from '../../atoms/Button/Button';
 import Icon from '../../atoms/Icon/Icon';
 import { faEnvelope, faPhone } from '@fortawesome/free-solid-svg-icons';
-
 const HelpWrap = styled.div`
   background: ${colors.white};
   box-sizing: border-box;
 `;
-
 const HelpContent = styled.div`
   margin: 0 auto;
   max-width: 700px;
 `;
-
 const FlexColCenter = styled(FlexCol)`
   text-align: center;
 `;
-
 const ButtonAsLink = styled.a`
   ${buttonStyle}
 `;
+interface DetailsProps {
+  [key: string]: {
+    telephone: {
+      label: string;
+      tel: string;
+    };
+    text: React.ReactNode;
+  };
+}
```

Comment:
Given we only have two options (borrowers and investors). I guess we could use the HelpLine enum to add a more strict typing in that key. So whenever we add another one it gets properly typed.

```suggestion
type DetailsProps = Record<HelpLine, {
    telephone: {
      label: string;
      tel: string;
    };
    text: React.ReactNode;
  }>;
```